app/helpers/container_build_helper/textual_summary.rb

```diff
@@ -0,0 +1,101 @@
+module ContainerBuildHelper::TextualSummary
+  #
+  # Groups
+  #
+
+  def textual_group_properties
+    %i(name creation_timestamp resource_version)
+  end
+
+  def textual_group_relationships
+    %i(ems container_project )
+  end
+
+  def textual_group_smart_management
+    items = %w(tags)
+    items.collect { |m| send("textual_#{m}") }.flatten.compact
+  end
+
+  def textual_group_build_instances
+    {
+      :additional_table_class => "table-fixed",
+      :labels                 => [_("Name"), _("Phase"),
+                                  _("Message"), _("Reason"),
+                                  _("Pod"), _("Output Image"),
+                                  _("Start Timestamp"), _("Completion Timestamp"),
+                                  _("Duration"),
+                                 ],
+      :values                 => collect_build_pods,
+    }
+  end
+
+  def collect_build_pods
+    @record.container_build_pods.collect do |build_pod|
```

Comment:
just a thought - when I am reading the build_pod term, it sounds like a method rather than an object, because of the build verb. took me couple of times to reread it differnently. perhaps pod_build?